cmake/Modules/FindLibClang.cmake

```diff
@@ -53,14 +54,14 @@ if(NOT LIBCLANG_LDFLAGS)
     # -rpath is relative path for all linked libraries.
     # The second "." is argument to rpath.
 
-    if(UNIX)
-        set(LIBCLANG_LDFLAGS_OS "-L--enable-new-dtags -L-rpath=.")
-    elseif(APPLE)
-        set(LIBCLANG_LDFLAGS_OS "-L-rpath=${LIBCLANG_DIR_PATH}")
+    if(UNIX AND NOT APPLE)
```

Comment:
UNIX is always true on MacOSX?
From what I can see it should have equivalent behavior if the checks are flipped, first apple then unix. The "AND..." can be removed